### PR TITLE
LPS-106824 Provide endpoint to retrieve views

### PR DESCRIPTION
**Pull request by dgarciasarai**
- Mock data until Analytics Cloud endpoints are available
- First action command that will call Analytics Cloud, there will be more in a few days

### CODE REVIEW

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetAnalyticsReportsTotalViewsMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+public class GetAnalyticsReportsTotalViewsMVCActionCommand
+	extends BaseMVCActionCommand {
+
+	@Override
+	protected void doProcessAction(
+			ActionRequest actionRequest, ActionResponse actionResponse)
+		throws Exception {
+
+		JSONObject jsonObject = JSONUtil.put(
+			"analyticsReportsTotalViews", 9999);
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetAnalyticsReportsTotalViewsMVCActionCommand.java]: Mock

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**